bookkeeper-common/src/main/java/org/apache/bookkeeper/common/util/OrderedExecutor.java

```diff
@@ -526,7 +520,7 @@ public boolean preserveMdc() {
      * @param orderingKey
      * @param r
      */
-    public void executeOrdered(Object orderingKey, SafeRunnable r) {
+    public void executeOrdered(Object orderingKey, Runnable r) {
```

Comment:
I am pretty sure that this API is uses by other projects as utility.

Could we still keep the old methods in order to keep compatibility?

Reply:
`SafeRunnable` interface is not being removed and changing these methods to require `Runnable` should be ok since it's the "parent" interface.

Reply:
eg: passing a `SafeRunnable` instance is still valid, because it implements `Runnable`.